src/components/timeline/TimelineViewControls.svelte

```diff
@@ -69,22 +69,28 @@
     scrollIfOffscreen();
   }
 
-  $: if (
-    typeof window !== 'undefined' &&
-    ($selectedActivityDirective || $selectedSpan || $simulationDatasetId !== null)
-  ) {
-    viewURL = new URL(window.location.href);
-    viewURL.searchParams.set(SearchParameters.START_TIME, new Date(viewTimeRange.start).toISOString());
-    viewURL.searchParams.set(SearchParameters.END_TIME, new Date(viewTimeRange.end).toISOString());
-    if ($selectedActivityDirective) {
-      viewURL.searchParams.set(SearchParameters.ACTIVITY_ID, $selectedActivityDirective.id.toFixed());
-    }
-    if ($selectedSpan) {
-      viewURL.searchParams.set(SearchParameters.SPAN_ID, $selectedSpan.span_id.toFixed());
-    }
-    if ($simulationDatasetId) {
-      viewURL.searchParams.set(SearchParameters.SIMULATION_DATASET_ID, $simulationDatasetId.toFixed());
+  // The time range can potentially become invalid by zooming out too much which will cause the following to fail
```

Comment:
Zooming out or in? I think there's an issue tracking an error when zooming into too far. But wondering if we should be looking for an invalid time range closer to the source?

Reply:
Sorry! Yes, zooming "in" too much.

Reply:
Got it. I'll keep that other issue open but no harm in try/catching this.